src/test/run-make/print-rustc-path/Makefile

```diff
@@ -0,0 +1,9 @@
+-include ../../run-make-fulldeps/tools.mk
+
+ifdef IS_WINDOWS
+all:
+	$(RUSTC) -Zunstable-options --print rustc-path | $(CGREP) bin\rustc
```

Comment:
1. Looks like `\` needs to be escaped here.
2. FWIW, `CGREP` supports regexes to match on both `\` and `/` together.